### DISCUSSION TITLE
Just a thought on Ethereum

### DISCUSSION THREAD

**Discussion by joaquinbarrientoscr** [Cyfrin/foundry-full-course-cu#1478] [General]
What are the requirements for the Ethereum blockchain to create a new block?
 i.e. when a block reaches 100 txns, it is time to create a new block for the 101th.

**Comment by EngrPips** [comment]
Hello @jbmkahdeksan , i think this differ from blockchain to blockchain. but honestly it is more of the time range required for creation of a new block which the mining process is a key determinant of  than the number of transaction. for example on ethereum blockchain i think a new block is created every 12-15s. But honestly to understand deeply any blockchain and how they work you need to do a lot of studying and researching by yourself by reading the blockchain whitepaper, yellow paper and every resources that gives detailed information about the blockchain. no tutorial will ever teach you everything

**Reply by joaquinbarrientoscr** [reply]
Thanks for the response and the encouragement to look broader I appreciate it, it is an important part of the path of growth!
